app/models/daily_summary_report.rb

```diff
@@ -116,6 +116,16 @@ def cpc
     gross_revenue / clicks_count
   end
 
+  def publisher_cpm
+    return Money.new(0) unless paid_impressions_count > 0
+    property_revenue / (paid_impressions_count / 1000.to_f)
+  end
```

Comment:
Would like to see a descriptive comment on this one to prevent confusion for future contributors.